### PR TITLE
Register choding.is-a.dev

### PR DESCRIPTION
**Pull request by seldomgreeen**
Register choding.is-a.dev with CNAME record pointing to choding.is-a.dev.

### CODE REVIEW

File: domains/seldomgreen.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "seldomgreeen",
+           "email": "72107155+seldomgreeen@users.noreply.github.com",
+           "discord": "761646485334523954"
+        },
+    
+        "record": {
+            "CNAME": "is-a.dev"
```
Comment (CuteDog5695) [inline domains/seldomgreen.json:9]: ?

File: domains/choding.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "seldomgreeen",
+           "email": "",
+           "discord": "761646485334523954",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.cgki5N5OYOlJmnAYS6DzoIrdJ-1vcd3n2t9cL445Vd2Y-aBbfbrgYNHMrQos494j3ZLjn96bS-9aX5dYjCSOm6GsKENLGGvswH-Qh5znCNVj8Md0ibMrqNi96lmn80Yb8s5xBf-jXllxssis2Y1FxoRz_MhkjfvWDnoZnMBYz9rFF7I9gdmbjVRC6St4N6kSRsCm2TH4u4ubqEF2Zw8pLBJYhvoQvQTQTZdUiHG29ySyZn2eMG8PAdRDk7BmXcsL0VdeHWSVHPGvZtG9u31KcNg6wE9hw8VbKvSP5Vf6TnQnoem5bCPkmHmtQFPDcfdYed6CtBEsSfysG2BSmqWoOQ.guHTLOLVPe50QKteOYXHjg.KtWS1XgBBwK12FCcRyPiCTNPk0VwIE5iujNLDNwj2rQoBrRbOY083Bpm-YGbWGa64206pD1j6uSGPaWoGERKW0525UPda_gH7pIHhaYP8dWNVlEJomUClv6EWN1dEOvYQvXm9djZCzXokOSUF3VOtQ.Sy2nbCwS7g13giYrA9wBYA"
+        },
+    
+        "record": {
+            "CNAME": "choding.is-a.dev"
```
Comment (CuteDog5695) [inline domains/choding.json:10]: You cannot point the domain to itself.

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/seldomgreen.json (hunk 1 above)
Comment (seldomgreeen) [reply domains/seldomgreen.json:9]: Yeah this change is fine 👍

Comment (CuteDog5695) [reply domains/seldomgreen.json:9]: Its not, you can't point your domain to our domain.

Comment (seldomgreeen) [reply domains/seldomgreen.json:9]: Mb, I'm gonna try to find somewhere to host and then I'll let you know what to update it to

**Pull request closed without merging**